src/scs/scs.md

```diff
@@ -1,3 +1,7 @@
 # SCS
 
+Supply Chain Security is a process of securitng the machinery of the development, build and release environment, in other words, securing every component that a software artifact might be touching on its way from the developer towards the consumer. The software artifact should be encrypted on each possilbe transition phase and the integrity of it should be verified at each destination.
 
+The SBOM (Software Bill Of Materials) containing reference to each dependency, its source and version togerher with provenance, containing build information are collected at the build time, signed and used for vulnerability analysis during the next steps. The SBOM is also published together with the final image, thus making further analysis of the software possible by the end customer.
```

Comment:
1. In the first paragraph, you tell about the SCS for the first time. Now we know what is it. How did you go from SCS to SBOM? What is the connection?
2. Misspelling: togerher -> together

Reply:
will try to come up with a couple of sentences to connect those 2.